ontology/yaml/resources/HVAC/entity_types/FAN.yaml

```diff
@@ -482,6 +482,16 @@ FAN_SS_VSC_SCRSS:
   - VSC
   - SCRSS
 
+FAN_SFSS_SFVSC_SSPC_FDPSM:
+  description: "Fresh air fan with supply air static pressure sensor based VFD Modulation."
+  is_canonical: true
+  implements:
+  - FAN
+  - SFSS
+  - SFVSC
+  - SSPC
```

Comment:
Does this fan serve a single space? Or does it have downstream ductwork? I would not expect a standalone fan to have SSPC

Reply:
This Fresh air Fan is supplying fresh air to the AHU Room from there AHU takes the fresh Air and supply it to VAVs.

Reply:
Yes there are

Reply:
Sensor is present in 2/3 of duct from the supply plenum.